lib/src/res/strings/tr.dart

```diff
@@ -233,14 +233,15 @@ Map<String, String> tr = {
   "UA": "Ukrayna",
   "AE": "Birleşik Arap Emirlikleri",
   "GB": "Birleşik Krallık",
-  "ABD": "Amerika Birleşik Devletleri",
```

Comment:
As I mentioned in https://github.com/Daniel-Ioannou/flutter_country_picker/discussions/59, this had translated wrong.

Reply:
Fixed issue #63